src/app/components/question-management/question-management.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="wrapper">
+  <div *ngIf="isDataExists(); else emptyList">
+    <div class="list-of-cards" *ngFor="let card of questionCards.listOfQuestions; index as idx">
```

Comment:
better to call index as index or i

Reply:
also better to add **trackBy** function as you modify array and elements can be deleted, which causes indexes chage